openmrs-19/src/test/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceTest.java

```diff
@@ -111,9 +111,16 @@ public void shouldCreatePatientWithGivenParameters() {
         doReturn(Collections.singletonList(location)).when(locationService).getLocations(location.getName());
 
         openMRSActionProxyService.createPatient(person.getPreferredName().getGivenName(), person.getPreferredName().getMiddleName(),
-                person.getPreferredName().getFamilyName(), person.getPreferredAddress().getAddress1(), new DateTime(person.getBirthdate()),
-                person.getBirthdateEstimated(), person.getGender(), person.getDead(), causeOfDeath.getUuid(), patient.getMotechId(),
-                location.getName(), identifiersMap);
+                person.getPreferredName().getFamilyName(), person.getPreferredAddress().getAddress1(),
```

Comment:
to simplify this code, maybe use new object to get PreferredAddress and then use getters on it